rook/README.md

```diff
@@ -77,9 +77,24 @@ $ kubectl create namespace kubestone
 
 ### `iperf`
 ```bash
-$ popper run -f workflows/iperf.yml
+# between server and client-1
+$ popper run -f workflows/iperf.yml -s _SERVER=<server-hostname> -s _CLIENT=<client-one-hostname>
+
+# between server and client-2
+$ popper run -f workflows/iperf.yml -s _SERVER=<server-hostname> -s _CLIENT=<client-two-hostname>
+
+# between server and client-3
+$ popper run -f workflows/iperf.yml -s _SERVER=<server-hostname> -s _CLIENT=<client-three-hostname>
+.
+.
 ```
 
+The hostnames to be provided to the substitution variables of the `popper run` command should be the `kubernetes.io/hostname` value of the nodes in the cluster.
```

Comment:
can we change this to say that "this is the the hostname or IP of the host that is running within a kubernetes cluster". This with the goal of avoiding having people visit that URL (which doesn't exist BTW).

Reply:
Yeah